rapids_dask_dependency/patches/distributed/comm/__rdd_patch_ucx.py

```diff
@@ -561,6 +562,17 @@ def get_connector(self):
         return UCXConnector()
 
     def get_listener(self, loc, handle_comm, deserialize, **connection_args):
+        warnings.warn(
```

Comment:
https://docs.python.org/3/library/textwrap.html#textwrap.dedent can be used to format this a bit nicer for the source code and users.

Reply:
Done in https://github.com/rapidsai/rapids-dask-dependency/pull/116/commits/1ac9a01d4a31fa5ff849b91229619726f868d441.